strawberry/tools/create_type.py

```diff
@@ -50,15 +51,19 @@ def hello(info) -> str:
         if not isinstance(field, StrawberryField):
             raise TypeError("Field is not an instance of StrawberryField")
 
-        if field.python_name is None:
+        field_name = field.python_name or (
+            to_snake_case(field.graphql_name) if field.graphql_name else None
```

Comment:
mmh, not sure I like this 🤔

Reply:
I'm not sure either. Basically, we need a unique field name for a field created like this:

```python
first_name = strawberry.field(name="firstName")
```

We could either:
1. go with the provided `name` (i.e., `firstName`)
2. convert the provided `name` to snake case (i.e., `first_name`)
3. or add another parameter to `strawberry.field` (e.g., `python_name`)

Reply:
Good idea, just done that. Thanks for the review!